plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -243,10 +246,12 @@
     private KVMStoragePoolManager _storagePoolMgr;
 
     private VifDriver _defaultVifDriver;
+    private VifDriver _tungstenVifDriver;
```

Comment:
Can you please remove all occurrences of underscore `_` at the beginning of the variable name?
Many variables on the codebase have `_` at the beginning but it is not aligned with the [Java variable naming convention](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html).

According to [Java variable naming convention](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html):

> A similar convention exists for the underscore character; while it's technically legal to begin your variable's name with "_", this practice is discouraged. White space is not permitted.

Reply:
It is done, but to me, this "_" is like a special rule of Cloudstack when I first read Cloudstack code. And I'm pretty sure that 90% of newcomers will have the same thinking as I do

Reply:
@huyle-uni I totally understand your point, we have too much of these "_" in the codebase from the old days and this does not make it easy for newcomers.

Thanks for addressing the review :+1:.